js/app/restaurant/parseOpeningHours.jsx

```diff
@@ -0,0 +1,55 @@
+import moment from 'moment'
+import _ from 'lodash'
+
+const locale = $('html').attr('lang')
+
+moment.locale('en')
```

Comment:
Is this needed by `moment.weekdaysMin()`?

Reply:
Yes we need to set the locale in english to build the object's key, because `openingHours` in DB are in english.